.eslintrc

```diff
@@ -35,6 +35,7 @@
     "react/jsx-first-prop-new-line": 0,
     "import/prefer-default-export": 0,
     "react/jsx-filename-extension": 0,
+    "react/jsx-props-no-spreading": 0,
```

Comment:
codeFactor always complians this rule. 
But it's quite common coding style in current codebase. Disable this rule.

<img width="1147" alt="Screenshot 2024-07-09 at 2 20 05 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/3c1ac845-ebcf-46c0-8741-3887996eb46a">